__test__/integration/debt_token_api/runners/owner_of.ts

```diff
@@ -0,0 +1,54 @@
+// External
+import * as Web3 from "web3";
+import { BigNumber } from "bignumber.js";
+
+// Types
+import { ScenarioRunner, TestAPIs, TestAdapters } from "./";
+import { DebtTokenScenario } from "../scenarios";
+
+export class OwnerOfScenarioRunner extends ScenarioRunner {
+    public testScenario(
+        scenario: DebtTokenScenario.OwnerOfScenario,
+        web3: Web3,
+        testAPIs: TestAPIs,
+        testAdapters: TestAdapters,
+    ) {
+        const { debtTokenAPI, orderAPI } = testAPIs;
+
+        let tokenIDs: BigNumber[] = [];
+
+        describe(scenario.description, () => {
+            beforeEach(async () => {
+                for (let loanOrder of scenario.orders) {
+                    const order = await this.generateDebtTokenForOrder(
```

Comment:
I have almost the exact same beforeEach in my test suite -- let's create a helper in the `ScenarioRunner` class for `generateDebtTokensForOrders` that takes in the `scenario.orders` and returns an array of tokenIDs.  Will help reduce repetition.